### PR TITLE
Adding Navigation component with search and footer

### PR DESCRIPTION
**Pull request by asimqt**

### CODE REVIEW

File: app/isomorphic/components/layouts/footer/index.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import './styles.m.css';
+ const Footer = (props) => (
+    <div styleName="footer"> Footer </div>
+)
```
Comment (asimqt) [inline app/isomorphic/components/layouts/footer/index.js]: Map menu group FooterLinks here.

File: app/isomorphic/components/layouts/footer/index.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      isExternalLink: PT.bool,
+      completeUrl: PT.string,
+      title: PT.string
+    })
+  )
+};
+
+export const Footer = connect(
+  mapStateToProps,
+  () => ({})
```
Comment (nandap4790) [inline app/isomorphic/components/layouts/footer/index.js]: if `mapDispatchToProps` is not there, you could also just have it as `null`

File: app/isomorphic/components/layouts/footer/styles.m.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
+.footer {
+  clear: both;
+  text-align: center;
+  font-size: 20px;
+  background: black;
```
Comment (nandap4790) [inline app/isomorphic/components/layouts/footer/styles.m.css]: `background` to `background-color`

File: app/isomorphic/components/layouts/footer/styles.m.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+  color: white;
```
Comment (nandap4790) [inline app/isomorphic/components/layouts/footer/styles.m.css]: why not variables ?

File: app/isomorphic/components/layouts/footer/styles.m.css [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+  padding: 20px 0;
+}
+
+.link {
+  color: #fff;
```
Comment (nandap4790) [inline app/isomorphic/components/layouts/footer/styles.m.css]: Here as well

File: app/isomorphic/components/navigation-component/navbar-search/index.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+    });
+  }
+
+  render() {
+    let formStyle = {
+      transform: this.state.isSearchFormOpen ? "translate(0, 0)" : ""
+    };
+    let initialStyle = {
+      opacity: this.state.initialized ? 1 : 0
+    };
```
Comment (nandap4790) [inline app/isomorphic/components/navigation-component/navbar-search/index.js]: why not a const ?

File: app/isomorphic/components/layouts/footer/styles.m.css (hunk 4 above)
Comment (asimqt) [reply app/isomorphic/components/layouts/footer/styles.m.css]: These styles are very skeleton-ish and will be replaced by actual styles. And we are going to add the CSS architecture changes which we're following in Vikatan after once we're confident on it.

File: app/isomorphic/components/navigation-component/navbar-search/index.js (hunk 6 above)
Comment (asimqt) [reply app/isomorphic/components/navigation-component/navbar-search/index.js]: changed.

**Pull request merged**